src/index.html

```diff
@@ -21,7 +21,368 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="index.html"
```

Comment:
All navigation links point to 'index.html'. If these links are intended to navigate to different pages or sections, ensure the 'href' attributes are set correctly.